audio_context.html

```diff
@@ -0,0 +1,321 @@
+<!doctype html>
+<head>
+  <title>Web Audio Api - Audio Context</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+
+<style>
+
+  :root {
+    --flex-direction: column;
+    --line-height: 4em;
+  }
+
+  @media(min-width: 250px) {
+    :root {
+      --flex-direction: row;
+      --line-height: 8em;
+    }
+  }
+
+  h1 {
+    text-align: center;
+  }
+
+  main {
+    outline: thick double;
+    background-color: whitesmoke;
+  }
+
+  fieldset {
+    display: flex;
+    border: none;
+    background: yellow;
+  }
+
+  form > fieldset {
+    flex-direction: var(--flex-direction);
+  }
+
+  fieldset fieldset {
+    background: red;
+  }
+
+  fieldset input[type="range"] {
+    width: 20vw;
+    display: inline-block;
+    top: 0.3em;
+    position: relative;
+  }
+
+  form > fieldset > label {
+    background: turquoise;
+    line-height: var(--line-height);
+  }
+
+  form > header {
+    background: yellow;
+  }
+
+  [type="checkbox"] {
+  }
+
+</style>
+
+</head>
+
+<body>
+
+  <header>
+    <h1>DevPunks Sequencer</h1>
+  </header>
+
+<main>
+
+  <form action=#>
+
+  <header>
+    DevPunks Sequencer
+
+    <label>
+      BPM
+      <input type=range min=10 max=522 value=251>
+    </label>
+
+    <label>
+      Play
+      <input name="Play" type="checkbox">
+    </label>
+  </header>
+
+  <fieldset>
+
+    <legend>Sweep</legend>
+
+    <label>
+      Attack
+      <input type="range" name="attack" min="0" max="1" value="0.2" step="0.1">
+    </label>
+
+    <label>
+      Release
+      <input type="range" name="release" min="0" max="1" value="0.5" step="0.1">
+    </label>
+
+    <fieldset>
+
+      <legend hidden>Checkboxes</legend>
```

Comment:
Wheb you figure out a better legend text you want to keep that content buy make it _"invidible"_ not `hidden`. Fine for now to yank it out the dom but remember for later.  Ultimately there should be no `hiddwn`s unless controlled by javascript.  But fine to use during development. Just remember to remove loke console.log.

Reply:
Sounds good ,I hadn't thought about using display:none.Gracias